src/asphalt/core/_concurrent.py

```diff
@@ -98,6 +107,15 @@ class TaskFactory:
     exception_handler: ExceptionHandler | None = None
     _finished_event: Event = field(init=False, default_factory=Event)
     _task_group: TaskGroup = field(init=False)
+    _tasks: set[TaskHandle] = field(init=False, default_factory=set)
+
+    def all_task_handles(self) -> set[TaskHandle]:
```

Comment:
Maybe more intuitive?
```suggestion
    def get_all_tasks(self) -> set[TaskHandle]:
```

Reply:
This was what I modeled it after: https://docs.python.org/3.11/library/asyncio-task.html#asyncio.all_tasks

Reply:
Ok, I guess it's fine then :+1: